src/renderer.ts

```diff
@@ -89,14 +90,17 @@ export class Renderer {
     return {status: statusCode, content: result};
   }
 
-  async screenshot(url: string): Promise<Buffer> {
+  async screenshot(url: string, options?: object): Promise<Buffer> {
     const page = await this.browser.newPage();
 
-    await page.goto(url, {waitUntil: 'networkidle0'});
+    page.setViewport({width: 1000, height: 1000});
 
-    // Typings are out of date.
-    const buffer =
-        await page.screenshot({type: 'jpeg', encoding: 'base64'} as any);
+    await page.goto(url, {timeout: 10000, waitUntil: 'networkidle0'});
+
+    // Must be jpeg & binary format.
```

Comment:
Why must it be jpeg and not png?

Reply:
In the past we had lots of issues with png due to image size if you try to take a really large screenshot.